docs/environment-and-software/user-environment/package-managers/pixi.md

```diff
@@ -0,0 +1,197 @@
+# Using `Pixi` for Package Management
+
+NCAR system users can also use [**`Pixi`**](https://pixi.sh/latest/), a fast package manager built in Rust that uses the conda ecosystem. 
+Pixi provides a modern, project-based workflow for
+managing dependencies across Python, R, C/C++, Julia, and other
+languages, with significantly faster performance than traditional conda.
+
+Pixi builds upon the foundation of the Conda ecosystem, introducing a workspace-centric approach rather than focusing solely on environments.
+This project workspace approach offers a more organized and efficient way to manage dependencies and run code, tailored to modern development practices.
+
+!!! note "uv vs. Pixi vs. Conda"
+    Pixi uses the same package repositories as Conda (conda-forge, etc.)
+    but with much faster dependency resolution and environment creation. If
+    you currently use Conda and find it slow, Pixi may be an excellent
+    alternative. For pure Python projects, consider `uv`
+    instead.
+
+
+## Using `pixi` on NCAR Systems
+
+Pixi is available as an environment module on NCAR systems.
+
+Before using `pixi`, ensure no conda environments are activated by running:
+```bash
+conda deactivate
+```
+
+Then, load the `pixi` module:
+```bash
+module load pixi
+```
+
+The module will automatically unload the system Conda modules to prevent conflicts.
+
+This module:
+
+- provides access to the `pixi` command.
+- configures Pixi to store cache and package data in your
+  `/glade/work` space to avoid filling your home directory quota.
+- allows you to create fast, multi-language environments for your research.
+- supports reproducible workflows through automatic lock file generation.
+
+!!! note "Cache location"
```

Comment:
This feels a bit redundant with the bullet point above. Perhaps an independent point could be made here instead by just saying something like:

"By default, Pixi will place its cache in your home directory, which can quickly exceed its quota as Pixi downloads packages. Pixi allows you to change the location of the cache. Our `pixi` module does this for you, setting the cache to a path in your `/glade/work/$USER` directory."

and then the bullet point above could just read:

- configures Pixi to store cache and package data in your `/glade/work/$USER` space